package/MDAnalysis/analysis/hbonds/hbond_analysis.py

```diff
@@ -416,7 +416,7 @@ class HydrogenBondAnalysis(object):
     #: Use the keyword *acceptors* to add a list of additional acceptor names.
     DEFAULT_ACCEPTORS = {
         'CHARMM27': tuple(set([
-            'O', 'OH2', 'OW', 'OD1', 'OD2', 'SG', 'OE1', 'OE1', 'OE2', 'ND1', 'NE2', 'SD', 'OG', 'OG1', 'OH'])),
+            'O', 'OC1', 'OC2', 'OH2', 'OW', 'OD1', 'OD2', 'SG', 'OE1', 'OE1', 'OE2', 'ND1', 'NE2', 'SD', 'OG', 'OG1', 'OH'])),
```

Comment:
Technically we should have separate Amber parameters but the way we have been doing this is to lump pretty much everything into CHARMM27. As long as you have a comment in the table above it should be fine.